webapp/src/components/LoginModal/LoginWithFacebook.js

```diff
@@ -22,23 +24,21 @@ const useStyles = makeStyles((theme) => ({
 const LoginWithFacebook = ({ onSubmit }) => {
   const classes = useStyles()
 
-  const responseFacebook = (response) => {
+  const facebookResponse = (response) => {
     onSubmit(true, response.profile.email, response.profile.id)
   }
 
   const handleError = (error) => {
-    if (error) {
-      onSubmit(false, "", "")
-    }
+    if (error) { onSubmit(false, "", "") }
```

Comment:
When the validation has only one line in its body, you can remove curly braces. So, the result is `if (error) onSubmit(false, "", "")`